test/functional/feature_fee_estimation.py

```diff
@@ -290,6 +291,45 @@ def sanity_check_rbf_estimates(self, utxos):
         est_feerate = node.estimatesmartfee(2)["feerate"]
         assert_equal(est_feerate, high_feerate_kvb)
 
+    def test_old_fee_estimate_file(self):
+        # Get the initial fee rate while node is running
+        fee_rate = self.nodes[0].estimatesmartfee(1)["feerate"]
+
+        # Restart node to ensure fee_estimate.dat file is read
+        self.stop_node(0)
+        self.start_node(0)
```

Comment:
I think you can use `self.restart_node(0)` here

Reply:
Thanks fixed